DEVELOPMENT.md

```diff
@@ -28,7 +28,7 @@ Setup:
 1. Clone the repo to a directory of your choosing.
 1. Open a terminal in the repo root.
 1. Execute `git config --local include.path ./.gitconfig` to attach the repo-specific Git configuration.
-1. Execute `just install-tools` to install development tools.
+1. Execute `just install-tools` to install development tools (cargo-machete, cargo-nextest, release-plz, cargo-semver-checks, cargo-audit, cargo-hack, cargo-mutants, cargo-llvm-cov, cargo-sort-derives and nightly toolchain with miri component).
```

Comment:
This information is duplicative. We should not restate what this command does, as it only encourages things to go out of sync.